main.go

```diff
@@ -110,12 +119,23 @@ func main() {
 	if *o == "" {
 		*o = "."
 	}
+
 	dst := filepath.Join(*o, "xcode")
+	trace("remove %d", dst)
 	if err := os.RemoveAll(dst); err != nil {
-		panic(err)
+		log.Fatalln(err)
+
 	}
+
+	trace("rename %v to %v", dir, dst)
 	if err := os.Rename(dir, dst); err != nil {
-		panic(err)
+		log.Fatalln(err)
+	}
+}
+
+func trace(format string,  v ...interface{}) {
```

Comment:
`trace` should be called `tracef` since it accepts parameters/behaves like a printf, so that it can be compatible with `go vet`.

See https://godoc.org/golang.org/x/tools/cmd/vet#hdr-Printf_family.

Specifically,

> If the function name ends with an 'f', the function is assumed to take a format descriptor string in the manner of fmt.Printf. If not, vet complains about arguments that look like format descriptor strings.